### ISSUE TITLE
Can't change collectTypes

### ISSUE THREAD

**Issue by ggascoigne** [archfz/cypress-terminal-report#50]
Using version "2.0.0"

From the docs I'd expect that the following config code is correct,

```
  require('cypress-terminal-report/src/installLogsPrinter')(on, {
    collectTypes: ['cons:log', 'cons:info', 'cons:warn', 'cons:error'],
  })
```

But when I use that config I get this error.

```
 Error: [cypress-terminal-report] Invalid plugin install options:
=> .collectTypes: Additional properties not allowed
```

**Comment by ggascoigne** [comment]
Gah, didn't realize that there were two separate config options.

Can I suggest that you put a larger break in the docs between "Options for the plugin install" and "Options for the support install".  I've read that several time, and this is the first time I noticed that it's documenting two separate things.

**Comment by archfz** [comment]
Released in 2.1.0.
